src/compat/text.rs

```diff
@@ -161,12 +161,14 @@ where
     for line in reader.lines() {
         let line =
             line.map_err(|e| ErrorKind::io_error("Cannot read line from embedding file", e))?;
-        let mut parts = line.split_whitespace();
+        let mut parts = line
+            .split(|c| c == ' ' || c == '\t')
```

Comment:
maybe `.split(char::is_ascii_whitespace)` ?

Reply:
Unfortunately `is_ascii_whitespace` takes a `&char`, we get `char` here. So compiling fails with:

```
    |
165 |             .split(char::is_ascii_whitespace)
    |              ^^^^^
    |              |
    |              expected signature of `fn(char) -> _`
    |              found signature of `for<'r> fn(&'r char) -> _`
    |
```